lib/composers/with_redux.js

```diff
@@ -0,0 +1,30 @@
+import compose from '../compose';
+
+export default function composeWithRedux(fn, L, E, options) {
+  const onPropsChange = (props, onData) => {
+    const context = typeof props.context === 'function' ? props.context() : props.context;
```

Comment:
I see, that makes sense. When you say second argument, does the Loader
component go away then? That's very useful to override.

I think I'll need an example of passing props to the function at runtime,
not sure I understand that part.
On Fri, Jul 1, 2016 at 11:08 PM Arunoda Susiripala <notifications@github.com>
wrote:

> In lib/composers/with_redux.js
> <https://github.com/kadirahq/react-komposer/pull/93#discussion_r69371649>:
>
> > @@ -0,0 +1,30 @@
> > +import compose from '../compose';
> > +
> > +export default function composeWithRedux(fn, L, E, options) {
> > +  const onPropsChange = (props, onData) => {
> > +    const context = typeof props.context === 'function' ? props.context() : props.context;
>
> Here's the problem in finding the store. Getting it from the options works
> fine for testing. But, we can have that.
> But, the getting it from the context is something common for mantra, so
> this won't suite here.
> So,here's my suggestion.
>
> Let's pass a function as the second argument. If that's an object, we can
> consider it as the store. If it's a function, at runtime we'll pass props
> to it and that should return the store.
> What do you think?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kadirahq/react-komposer/pull/93/files/4b3fd07daad5732810c89ba066d7f0258f4cf6fd#r69371649>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAarJdu-6qjSRy-qEIuQFq5sN4RzMJ0hks5qRdYagaJpZM4JDVML>
> .
>